manifests/20-crd-tuned.yaml

```diff
@@ -35,9 +35,6 @@ spec:
             spec:
               description: 'spec is the specification of the desired behavior of Tuned. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#spec-and-status'
               type: object
-              required:
-                - profile
```

Comment:
By having those fields optional it might affect the way PAO present and track the tuned profile's status? https://github.com/openshift-kni/performance-addon-operators/blob/39c74c9b999b57d810934fe0913e8a64eaff162c/controllers/status.go#L240

Reply:
Thank you for looking @Tal-or !  But you're querying [Profile](https://github.com/openshift/cluster-node-tuning-operator/blob/master/manifests/20-crd-profile.yaml) in your code and not [Tuned](https://github.com/openshift/cluster-node-tuning-operator/blob/master/manifests/20-crd-tuned.yaml) that is modified here.  How will this cause problems to PAO?

Reply:
My mistake. I thought that the profile under Tuned is this [Profile](https://github.com/openshift/cluster-node-tuning-operator/blob/master/manifests/20-crd-profile.yaml)